k8s/application/application-deployment.yaml

```
@@ -0,0 +1,30 @@
+          resources:
+            requests:
+              memory: "512Mi"   # 최소 보장 메모리
+              cpu: "250m"
+            limits:
+              memory: "1024Mi"  # 최대 허용 메모리
+              cpu: "500m"
+          envFrom:
+            - configMapRef:
+                name: crm-config
+          ports:
+            - containerPort: 8080
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)